arch/arm64/boot/dts/marvell/accton-as4560-52p.dts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Device Tree For Accton as4560-52p.
+ *
+ * Copyright (C) 2021 Marvell
+ *
+ */
+
+#include "accton-as4560-52p.dtsi"
+
+/ {
+	model = "Marvell AC5X RD board";
+};
+
+&mdio {
+	status = "disabled";
+	pinctrl-names = "default";
+	phy0: ethernet-phy@0 {
+		reg = <0x0>;
+	};
+};
+
+&eth0 {
+	status = "disabled";
+	phy-mode = "sgmii";
+	phy = <&phy0>;
+};
+
+&usb0 {
+	status= "okay";
+};
+
+&usb1 {
+	compatible = "chipidea,usb2";
+	phys = <&usb1phy>;
+	phy-names = "usb-phy";
+	dr_mode = "host";
+};
+
```

Comment:
You should be describing all devices attached to the i2c buses here via

```
&i2c0 {
	...
}

&i2c1 {
	...
}
```

also describe the SFP ports as e.g. in https://github.com/dentproject/linux/blob/dent-linux-5.15.105/arch/arm64/boot/dts/marvell/ac5_db_trampoline.dts